docs/developer-guide.md

```diff
@@ -26,6 +26,30 @@ In this debug mode, `DotnetRunner` does not launch the .NET application, but wai
 
 Now you can run your .NET application with any debugger to debug your application.
 
+### Debugging UDF
+
+**Note that this is currenlty supported only on Windows.**
+
+Before running `spark-submit`, set the following environment variable:
+```shell
+set DOTNET_WORKER_DEBUG=1
+```
+Now, when you run your Spark application, a `Choose Just-In-Time Debugger` window will pop up. Choose a debugger.
+
+The debugger will break at the following location in `TaskRunner.cs`:
+```C#
+if (EnvironmentUtils.GetEnvironmentVariableAsBool("DOTNET_WORKER_DEBUG"))
+{
+    Debugger.Launch(); // <-- The debugger will break here.
+}
+```
+
+Now, navigate to the `cs` file that contains the UDF that you plan to debug, and set a breakpoint. (The breakpoint will say `The breakpoint will not currently be hit` because the worker hasn't loaded the assembly that contains UDF yet.)
```

Comment:
```suggestion
Now, navigate to the `.cs` file that contains the UDF that you plan to debug, and set a breakpoint. (The breakpoint will say `The breakpoint will not currently be hit` because the worker hasn't loaded the assembly that contains UDF yet.)
```